examples/graphql-api/README.md

```diff
@@ -0,0 +1,30 @@
+# GraphQL API
+
+This example demonstrates using the Fetch API to query a GraphQL endpoint. It also makes use of a [GraphQL Code Generator](https://www.graphql-code-generator.com/) which introspects the GraphQL API in order to generate types that can be used in our `useLoaderData` hooks for strong, generated types.
+
+## Setup
+
+- Setup the `GRAPHQL_API` environment variable
+  - See the [Remix docs](/docs/guides/envvars) for some suggested patterns
+- Run `$ npm install`
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/graphql-api)
+
+## Example
```

Comment:
Could this bit explain what's demonstrated in this example and what files to play close attention to?

Reply:
Thanks Kent, I've taken a stab at updating the README a bit, fresh eyes are appreciated though!